service-front/public/assets/v2/js/application.js

```diff
@@ -25666,6 +25666,12 @@ this["lpa"]["templates"]["shared.loading-popup"] = Handlebars.template({"compile
                 window.GOVUK.cookie('cookie_policy') || window.GOVUK.setDefaultConsentCookie()
             }
 
+            if (this.isInCookiesPage() && !this.isInIframe() && 'true' === window.GOVUK.cookie('submitted_cookie_page')) {
```

Comment:
I'm not sure what the purpose of this is. Could you please explain? Thanks.

Reply:
The conditional or the timeout below?
The conditional is set as the Cookie Consent section is checked on every page loaded & as I've created a very short-lived cookie to check if cookie options have been submitted. As it was causing further pain to see if we can just add a param to the URL.

The time out I've included as unless I included this it wouldn't run correctly locally.